cassandra/cqlengine/management.py

```diff
@@ -154,9 +154,10 @@ def _get_index_name_by_column(table, column_name):
     Find the index name for a given table and column.
     """
     protected_name = metadata.protect_name(column_name)
-    possible_index_values = [protected_name, "values(%s)" % protected_name]
+    possible_index_values = [protected_name, "values(%s)" % protected_name, "keys(%s)" % protected_name]
```

Comment:
Afaik there is also `entries(somemap)`, to sure if it's relevant for those tests.

Reply:
I thought that C* also has such indexes. Is the test broken there too?

Reply:
I didn't see it in the other PR.

But maybe I didn't get there yet

Reply:
Other PR?

Reply:
https://github.com/scylladb/python-driver/pull/339

Reply:
Oh ok, what I meant is that this is a possible index type in C* / Scylla. I don't know if it appears in tests.